src/OAINet.Node/Network/Node.cs

```diff
@@ -51,12 +60,25 @@ private void RegisterHandlers()
     }
     public async Task RunNode()
     {
-        _peer = InitializeNode(3024);
+        _peer = InitializeNode();
+        if (_peer == null)
+        {
+            _logger.LogInformation("the node cannot start.");
```

Comment:
Has the node an identifier ? Is it possible to add it to the log message ?

Reply:
euh no, is great like this.